src/audio/src/src.c

```diff
@@ -227,8 +227,9 @@ static int src_polyphase_init(struct polyphase_src *src, struct src_param *p,
 	 * use a simple copy function instead of 1 stage FIR with one
 	 * tap.
 	 */
+
 	n_stages = (src->stage2->filter_length == 1) ? 1 : 2;
-	if (src_in_fs[p->idx_in] == src_out_fs[p->idx_out])
+	if (p->in_fs[p->idx_in] == p->out_fs[p->idx_out])
```

Comment:
This is directly modifying the check added in commit 914c415eed214d413a30640181bb1dab04bc0445 , can you @singalsu check?

Reply:
This might be an issue in setup tables for src_lite since there some differences to normal. As far as I know they were hand-edited from some old version of normal SRC, while tables for normal SRC are all script generated. There's multiple ways to check for 1:1 conversion and this probably works for both src and src_lite. I'll test with normal SRC in testbench if this is OK, so it could be approved. I doubt anyone wants to do another hand edit round for src-lite. So please wait until I can check this. There's is no SOF CI coverage for this.

Reply:
To be more specific. There should be a script like this  for src-lite to be able to re-generate them if we would change anything into core functions: https://github.com/thesofproject/sof/blob/main/src/audio/src/tune/sof_src_ipc4_int32.m . Remove the rates combinations not needed and run the generator in Matlab or Octave.

Could you @tobonex please create a similar script and see if it would fix this issue? Without such script existing maintenance of src-lite is hard.

Reply:
a.so to remind, that src_lite was actually unused until https://github.com/thesofproject/sof/commit/7ee322f70a345ee476085b5e6b333263df812371

Reply:
@tobonex Maybe not familiar with Matlab? If you can make e.g. with spreadsheet a matrix of what in/our rates should be supported by src_lite, I can generate the new coefficients for you to test. I do see that possible in rates are 32, 44.1, and 48 kHz and possible out rates are 16  and 48 kHz. But are all in-out combinations supported?

Reply:
@singalsu I'm familliar with matlab. Not so familliar with SRC though , so I feel like I'm lacking some context. 
The thing I want to change here is still an issue I think : using hardcoded src table even when src_lite is used. Still, got octave running, I can try to create the script and check.

Reply:
@singalsu Added the generator for src_lite, and regenerated coef files. This by itself does not fix the issue, so the change in the if statement is still needed. However I'm guessing we still want the new files anyway so I added new commits.